src/doc/book/testing.md

```diff
@@ -499,6 +499,10 @@ be imported in every test with `mod common;`
 That's all there is to the `tests` directory. The `tests` module isn't needed
 here, since the whole thing is focused on tests.
 
+Note, when building integration tests, cargo will not pass the test flag to the
```

Comment:
Adding the grave accents and calling it an "attribute" instead of a "flag", since TRPL calls it an "attribute".